Snake/SnakeTest/Maze_Test.cs

```diff
@@ -11,7 +11,8 @@ public void IsTure_return_a_instance_of_type_of_2D_array_from_CreateMaze_functio
         {
             const int width = 10;
             const int height = 10;
-            BusinessLayer.Maze maze = new BusinessLayer.Maze(width, height);
+            BusinessLayer.Maze maze = new BusinessLayer.Maze(width, height
+                ,1);
```

Comment:
what's up with this dangling arg? Also I think you probably want to pass in the MazeLevel enum value rather than this integer.